spec/chromium-spec.js

```diff
@@ -275,7 +275,7 @@ describe('chromium feature', () => {
   })
 
   describe('navigator.geolocation', () => {
-    before(function () {
+    beforeEach(function () {
```

Comment:
why would this skip all tests ? wouldn't `before` be only applied to its `describe` block ?

Reply:
That's a great question, but without this change when I run `npm test -- --ci -g 'chromium feature'` i see 21 tests run. With this change, I see 82 tests run.

Reply:
... hm, i'm not seeing the same difference when running on CI. https://circleci.com/gh/electron/electron/162245 and https://circleci.com/gh/electron/electron/161864 show the same number of tests. So perhaps this is a weird interaction with `--grep` or something...?